dbfit-java/sqlserver/src/main/java/dbfit/environment/SqlServerEnvironment.java

```diff
@@ -80,15 +78,18 @@ protected String parseCommandText(String commandText) {
             String query) throws SQLException {
         DbParameterAccessorsMapBuilder params = new DbParameterAccessorsMapBuilder();
 
-        if (objname.contains(".")) {
-            String[] schemaAndName = objname.split("[\\.]", 2);
-            objname = "[" + schemaAndName[0] + "].[" + schemaAndName[1] + "]";
-        } else {
-            objname = "[" + normaliseName(objname) + "]";
+        objname = objname.replaceAll("[^a-zA-Z0-9_.#]", "");
+        String[] qualifiers = objname.split("\\.");
+        String bracketedName = "";
```

Comment:
So would be also be better to create constants for the `[` and `]` too in the method and use those in the loop instead of literals? Or is that overkill?